paclair/api/clair_requests_v3.py

```diff
@@ -43,5 +44,36 @@ def delete_ancestry(self, ancestry):
         raise PaclairException("Delete is not available for V3 api")
 
     def _iter_features(self, clair_json):
-        for feature in clair_json.get("ancestry", {}).get("features", {}):
+        for feature in clair_json.get("ancestry", {}).get("layers", {}):
             yield InsensitiveCaseDict(feature)
```

Comment:
You yield a layer instead of a feature. It changes the behavior of the method and breaks get_ancestry_statistics.